app/models/gobierto_data/dataset.rb

```diff
@@ -43,6 +57,14 @@ def available_formats
       [:csv]
     end
 
+    def dataset_path
```

Comment:
This method is not working fine

![Screenshot from 2021-04-23 05-16-27](https://user-images.githubusercontent.com/17616/115813223-16cf5900-a3f3-11eb-937b-d3385d9f21a1.png)

Reply:
(the / is missing!)